src/Form/Mixin/Value.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Kirby\Form\Mixin;
+
+/**
+ * @package   Kirby Form
+ * @author    Bastian Allgeier <bastian@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://opensource.org/licenses/MIT
+ */
+trait Value
+{
+	/**
+	 * @deprecated 3.5.0 Use `::toStoredValue()` instead
+	 * @todo remove when the general field class setup has been refactored
+	 */
+	public function data(bool $default = false): mixed
+	{
+		return $this->toStoredValue($default);
+	}
```

Comment:
`Question:` If we had deprecated this since 3.5 - maybe time to drop it now?

Reply:
The deprecation warning was wrong. We added that in the FieldClass but not in Field back then. I changed it to 5.0.0